### PR TITLE
:sparkles: Search for all files matching the pattern in the xml filepath

### PR DESCRIPTION
**Pull request by fabianvf**
This means if you look for `ejb-jar.xml` for example, it would also find `/config/ejb-jar.xml`, and allows you to put wildcards etc in the filepaths.

### CODE REVIEW

File: provider/builtin/provider.go [hunk 1]
```diff
@@ -181,9 +181,24 @@ func (p *builtinProvider) Evaluate(cap string, conditionInfo []byte) (lib.Provid
 			xmlFiles = append(xmlFiles, xhtmlFiles...)
 		} else if len(cond.XML.Filepaths) == 1 {
 			// Currently, rendering will render a list as a space seperated paths as a single string.
-			xmlFiles = strings.Split(cond.XML.Filepaths[0], " ")
+			patterns := strings.Split(cond.XML.Filepaths[0], " ")
+			for _, pattern := range patterns {
+				files, err := findFilesMatchingPattern(p.config.Location, pattern)
+				if err != nil {
```
Comment (shawn-hurley) [inline provider/builtin/provider.go:187]: This bit seems odd, if there is an error then we add the pattern as a file? Can we add a comment on why?

**Review by shawn-hurley — APPROVED** [review]
I think I understand that he matching patterns will return an error if it is a file path, and I assume that is what we are doing with the if/else?

Overall I think I am fine adding pattern searching to file paths input.

File: provider/builtin/provider.go (hunk 1 above)
Comment (fabianvf) [reply provider/builtin/provider.go:187]: my thinking was if something broke unintentionally it just goes back to the default behavior of trusting user input and running the rule

Comment (shawn-hurley) [reply provider/builtin/provider.go:187]: Yeah probably is the safest thing to do, I wonder can we add a debug log here I assume that we are going to be tracking down an obscure bug in the future :)

Comment (fabianvf) [reply provider/builtin/provider.go:187]: done

**Pull request merged**